site/en/tutorials/text/image_captioning.ipynb

```diff
@@ -874,17 +868,17 @@
         "        total_loss += t_loss\n",
         "\n",
         "        if batch % 100 == 0:\n",
-        "            print ('Epoch {} Batch {} Loss {:.4f}'.format(\n",
+        "            print('Epoch {} Batch {} Loss {:.4f}'.format(\n",
```

Comment:
If you're touching these it's always good to switch `.format` to f-string if you can.

Your `print("string", data)` are good too.